packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal-body.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
since these modal is related to the cashier, isn't it better to be located in the cashier package and get imported here ,like `AccountTransfer` modal